### ISSUE TITLE
Expected to stay on same plugin after selecting it and write more text

### ISSUE THREAD

**Issue by ido-ran** [KELiON/cerebro#129]
Please help me find better subject if you can.

# Issue
After writing some text the plugins show on the left panel, if I select another plugin and then write more text the plugin selection jumps back to the first instead of staying on the selected plugin and update the content panel.

# Steps to reproduce
1. Open Cerebro
1. Write few characters
1. Select another plugin, Translate for example
1. Write one more character

# Expected Behavior
Cerebro will stay on the same plugin and update the content panel

# Actual Behavior
Cerebro jupms back to the first plugin, Google search in my case.

**Comment by KELiON** [comment]
@ido-ran I think it makes sense! I'll try to fix it, should be super useful for plugins like translate or maps

**Comment by ido-ran** [comment]
I agree, and thank you for this project, I had to move from Mac to Windows and now finally at least I have search back.
